apps/src/feedback.js

```diff
@@ -183,7 +188,7 @@ FeedbackUtils.prototype.displayFeedback = function(
       continueText: options.continueText,
       isK1: options.level.isK1,
       freePlay: options.level.freePlay,
-      finalLevel: this.isFinalLevel(options.response)
```

Comment:
- [ ]  remove `isFinalLevel`

Reply:
re: removing `isFinalLevel` -- i recently added this helper because i didn't like how we had this expression sprinkled throughout this file:

https://github.com/code-dot-org/code-dot-org/blob/b34080dcdc025d78806c1c895b55c214e725c9f3/apps/src/feedback.js#L1986

but your change here uses `ScriptLevel#end_of_lesson?` instead -- do we no longer need to rely on the awkward "no more levels" response message from the server? could we remove it from ScriptLevelsHelper:

https://github.com/code-dot-org/code-dot-org/blob/6f8972c580059afe7e3c741fef4ee1214d82da8f/dashboard/app/helpers/script_levels_helper.rb#L30

Reply:
also, i'm 100% in favor of removing `isLastLevel` if we already have the equivalent info (which it seems like we do? it's really hard to tell with feedback.js code in its current state 😬)